File: ContactsDashBoard_Raghunandan/app/src/main/java/com/example/raghu/contactsdashboard_raghunandan/MyApplication.java
package com.example.raghu.contactsdashboard_raghunandan;

import android.app.Application;

/**
 * Created by raghu on 12/5/17.
 */

public class MyApplication extends Application {

    private static MyApplication application;


    @Override
    public void onCreate() {
        super.onCreate();
        if(application==null)
        {
            application = this;
        }
    }

    public static MyApplication getInstance() {
        return application;
    }


}

File: Pearson-Interview/app/src/main/java/raghu/me/myapplication/MyApplication.kt
package raghu.me.myapplication

import android.app.Application
import org.koin.android.ext.android.startKoin
import raghu.me.myapplication.network.applicationModule

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        startKoin(this,listOf(applicationModule))
    }
}